lib/index.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
```

Comment:
Why check this in? I'm not strictly opposed, just curious about the rationale.

Reply:
this is necessary if the user does not use the package management system.
to checkout directly from github.

Reply:
Okay. I'm not sure I really care to support that but I suppose it can't hurt.